test_communication/test/test_publisher_subscriber_python.py.in

```diff
@@ -0,0 +1,38 @@
+import sys
+from launch import LaunchDescriptor
+from launch.exit_handler import primary_exit_handler, default_exit_handler
+from launch.launcher import DefaultLauncher
+
+
+def test_publisher_subscriber():
+    ld = LaunchDescriptor()
+
+    ld.add_process(
+        cmd=[sys.executable,
+             '@TEST_PUBLISHER_PY_EXECUTABLE@',
+             '@TEST_MESSAGE_TYPE@',
+             '-r', '@TEST_RMW_IMPLEMENTATION_PY@'],
```

Comment:
This file looks to have only this one line significantly different than the non-python one it would be great to reuse. If we do that we could consider not duplicating the `multi_target` cmake macro to `custom_test_py` and instead extend the `multi_target` to do the cross product of the rmw implementations and the python ones as well.